docs/paper/dev/guides/pdc.md

```diff
@@ -0,0 +1,132 @@
+---
+slug: /dev/pdc
+---
+
+# Persistent Data Container (PDC)
+
+The Persistent Data Container (PDC) is a way to store custom data on a whole range of objects; such as items, entities, and blocks.
+The Full list of classes that support the PDC are:
+
+- [Chunk](#chunk)
+- [World](#world)
+- [Entity](#entity)
+- [TileState](#tilestate)
+- [ItemMeta](#itemmeta)
+- [Structure](#structure)
+
+## What is it used for?
+In the past, developers resorted to storing data within the NBT tags of an object. 
+This was a very hacky way of storing data, and was prone to breaking if the NBT tags were changed. 
```

Comment:
The last things is incorrect and it's always possible to break pdc if you change the nbt since pdc is stored in the nbttags. It's more about plugin conflict (with keys) than nbt change, the namespaced key resolve this issue